_includes/v21.2/sql/movr-statements-geo-partitioned-replicas.md

```diff
@@ -2,7 +2,7 @@
 
 The following examples use MovR, a fictional vehicle-sharing application, to demonstrate CockroachDB SQL statements. For more information about the MovR example application and dataset, see [MovR: A Global Vehicle-sharing App](movr.html).
 
-To follow along, run [`cockroach demo`](cockroach-demo.html) with the `--geo-partitioned-replicas` flag. This command opens an interactive SQL shell to a temporary, 9-node in-memory cluster with the [Geo-Partitioned Replicas Topology](../v21.2/topology-geo-partitioned-replicas.html) applied to the `movr` database.
+To follow along, run [`cockroach demo`](cockroach-demo.html) with the `--geo-partitioned-replicas` flag. This command opens an interactive SQL shell to a temporary, 9-node in-memory cluster with the `movr` database.
```

Comment:
Same question about this one, looks like the page exists and is not a redirect - curious why we are removing links in this PR?

I'm sure I'm missing something, let me know if so tho!

Reply:
Yep, same explanation.